backend/apps/resume/models.py

```diff
@@ -0,0 +1,84 @@
+from django.db import models
+from apps.candidates.models import Candidate
+from apps.vacancies.models import Vacancy
+from django.utils.translation import gettext_lazy as _
+from apps.resume.storages import OverwriteStorage
+
+
+def content_file_name(instance, filename):
+    file_path = "/".join(["cv", instance.owner.full_name, filename])
+    return file_path
```

Comment:
Rename the function to something like `genarate_file_name()` or add a docstring. 
Also you don't need to create a `file_path` variable if you use it only once:
```
return "/".join(["cv", instance.owner.full_name, filename])
```
I guess, there is no need to call `.join()` here, your list is not such large to do this extra joint, you can use f-string instead:
```
return f"cv/{instance.owner.full_name}/{filename}"
```